examples/operator/pf.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/sh
 
-kubectl port-forward --address localhost deployment/"$(kubectl get deployment --selector="planetscale.com/component=vtctld" -o=jsonpath="{.items..metadata.name}")" 15000 15999 &
```

Comment:
I pulled this from some planetscale documentation. We have to find out where that was and fix it there too.